### ISSUE TITLE
Add instance name to font label

### ISSUE THREAD

**Issue by RoelN** [arrowtype/type-x#75]
https://user-images.githubusercontent.com/45946693/68811490-c620d180-063e-11ea-814d-e5c74110c983.png

Also see https://github.com/arrowtype/type-x/pull/72#issuecomment-553642957

**Comment by arrowtype** [comment]
I think this is already working, right...? Or, is there something here that isn’t yet working as expected?

There might be opportunity to further improve the font override naming system, but I’ve made [a separate issue for that](https://github.com/arrowtype/type-x/issues/106).

**Comment by arrowtype** [comment]
I think this is already done – if you agree and there isn’t something you feel is still missing here, @RoelN, please feel free to close it.

**Comment by RoelN** [comment]
After some more testing, it works the first time the extension is turned on, but breaks after closing and re-opening the extension :-( So this should be investigated. (This can de done in parallel to the name override story)

**Comment by arrowtype** [comment]
No longer seems to be an issue. Nice!
